summer_meal_sites/function.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scriptFile": "__init__.py",
+    "bindings": [
+      {
+        "name": "mytimer",
+        "type": "timerTrigger",
+        "direction": "in",
+        "schedule": "1 1 1 4 *"
```

Comment:
Because functions run in UTC time zone running on first hour would mean we will be notified at very odd hours here in USA. To make it more readable I would suggest running it on first Sunday in April every year at 12 PM expression for which then becomes ` 0 0 12 1-7 Apr Sun` Can you please also update our wiki page here : 

https://github.com/ShelterApp/AddResources/wiki/Scraper-schedules

Reply:
Would "0 12 1 4 *" work?